src/pages/index.js

```diff
@@ -186,12 +304,32 @@ cardModalCloseBtn.addEventListener("click", () => {
 previewModalCloseBtn.addEventListener("click", () => {
   closeModal(previewModal);
 });
+deleteForm.addEventListener("submit", handleDeleteSubmit);
+avatarModalCloseBtn.addEventListener("click", () => {
+  closeModal(avatarModal);
+});
+deleteModalCloseBtn.addEventListener("click", () => {
+  closeModal(deleteModal);
+});
+avatarModalBtn.addEventListener("click", () => {
+  openModal(avatarModal);
+});
+profileAvatar.addEventListener("click", () => {
+  openModal(avatarModal);
+});
+
+document
+  .querySelector(".modal__submit-btn_cancel")
+  .addEventListener("click", () => {
+    const openedModal = document.querySelector(".modal_opened");
```

Comment:
`    const openedModal = document.querySelector(".modal_opened");
`

You don't need to search the opened modal because you know the `confirmation` modal. It's `deleteModal`


```js

document
  .querySelector(".modal__submit-btn_cancel")
  .addEventListener("click", () => {
    closeModal(deleteModal);
  });
````